src/test/java/unit/BqTableReferencedUpdate.java

```diff
@@ -49,14 +50,15 @@ protected void setupOnce() throws Exception {
         externalDataset.getDatasetId(),
         sharedExternalTable);
 
+    shareeUser = TestUser.chooseTestUserWhoIsNot(workspaceCreator);
+    shareeUser.login();
```

Comment:
This needs to come after `terra workspace add-user --email=shareeUser.email--role=WRITER`, otherwise this is `shareeUser` trying to invite themselves into the workspace

Reply:
makes sense, done